Core/NoOpRunner.Core/Iterators/LatestOrderIterator.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NoOpRunner.Core.Iterators
+{
+    class LatestOrderIterator : Iterator
+    {
+        private ShapeCollection collection;
+        private int position = -1;
+        private bool reverse = true;
+
+        public LatestOrderIterator(ShapeCollection collection, bool reverse = true)
```

Comment:
Why is is reversed by default?

Reply:
Because its following "LIFO" principle:
Data -> 1 2 3 4 5
LatestOrderIterator -> 5 4 3 2 1

Reply:
And why is it following "LIFO" principle, why do we want that for iterating over shapes?

Reply:
Because they want "3 different classes" to be implemented, so this is still something I guess